staging/src/k8s.io/noderesourcetopology-api/manifests/crd-instance-example.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: node1
+topologyPolicies:
+  - single-numa-node
+  - isolate-thread-policy
+zones:
```

Comment:
Who populates this ?

Reply:
node resource topology exporter, NodeFeatureDiscovery is planned to be such exporter. NFD authors are agree.

Reply:
Yup, we've agreed that NFD would provide this functionality.

Reply:
Resource Topology Exporter [here](https://github.com/swatisehgal/resource-topology-exporter) is a prototype for exposing topology information via CRDs. We were admised to move this functionality to NFD rather than a separate repository.
NFD Issue: Exposing Hardware Topology through CRDs in Node feature discovery can be found [here](https://github.com/kubernetes-sigs/node-feature-discovery/issues/333) and development on this in progress.

Reply:
That's correct.